apps/astro-party/src/pages/api/webhooks/castingwords.ts

```diff
@@ -17,7 +17,7 @@ const castingwordsWebhookReceiver = async (
     if (req.body.event === 'TRANSCRIPT_COMPLETE') {
       try {
         const {audiofile, order} = CastingwordsWebhookBody.parse(req.body)
-        await writeTranscriptToVideoResource(audiofile, order)
+        await writeTranscriptToVideoResource(audiofile, order, true)
```

Comment:
the `true` here is for anyplace where it's still portable text and hasn't been migrated to plain text. it needs to go away asap